plugins/cloudtrail/pkg/cloudtrail/extract.go

```diff
@@ -532,16 +532,14 @@ func getfieldStr(jdata *fastjson.Value, field string) (bool, string) {
 	case "ct.resources":
 		var resources string = ""
 		rlist := jdata.GetArray("resources")
-		if rlist == nil {
+		if len(rlist) == 0 {
```

Comment:
```suggestion
		if rlist == nil || len(rlist) == 0 {
```
As per `GetArray` documentation, it can still return `nil`:
> nil is returned for non-existing keys path or for invalid value type.

Reply:
Done, along with the version change.